packages/expo-keep-awake/src/ExpoKeepAwake.web.ts

```diff
@@ -1 +1,51 @@
-export default {};
+import { CodedError, Platform, Subscription } from 'expo-modules-core';
+
+import { KeepAwakeEvent, KeepAwakeEventState } from './KeepAwake.types';
+
+const wakeLockMap: Record<string, any> = {};
+
+/** Wraps the webWakeLock API https://developer.mozilla.org/en-US/docs/Web/API/Screen_Wake_Lock_API */
+export default {
+  async isAvailableAsync() {
+    if (Platform.isDOMAvailable) {
+      return 'wakeLock' in navigator;
+    }
+    return false;
+  },
+  async activate(tag: string) {
+    if (!Platform.isDOMAvailable) {
+      return;
+    }
+    const wakeLock = await (navigator as any).wakeLock.request('screen');
```

Comment:
I would rather go with polyfilling the type system with the missing typings. The very same way we do it in other modules:
https://github.com/expo/expo/blob/cd195fb2d1fd09d1d1ea8dad569065690b884a6e/packages/expo-cellular/src/ts-declarations/lib.dom.d.ts#L7-L18
and then no `as any` casting is happening:
https://github.com/expo/expo/blob/cd195fb2d1fd09d1d1ea8dad569065690b884a6e/packages/expo-cellular/src/ExpoCellular.web.ts#L20-L21